crates/graphql/src/schema/query_root.rs

```diff
@@ -779,6 +782,41 @@ impl QueryRoot {
             .map_err(Into::into)
     }
 
+    #[graphql(
+        description = "Returns featured collection NFTs ordered by market cap (floor price * number of NFTs in collection)",
+        arguments(
+            sort_by(description = "Choose sort for trending collections"),
+            order_direction(
+                description = "Arrange result in ascending or descending order by selected sort_by"
+            ),
+            limit(description = "Return at most this many results"),
+            offset(description = "Return results starting from this index"),
+        )
+    )]
+    async fn trending_collections(
+        &self,
+        context: &AppContext,
+        sort_by: Option<CollectionSort>,
+        order_direction: Option<OrderDirection>,
+        limit: i32,
+        offset: i32,
+    ) -> FieldResult<Vec<Collection>> {
+        let conn = context.shared.db.get().context("failed to connect to db")?;
+
+        let collections = queries::collections::trending(&conn, TrendingQueryOptions {
+            sort_by: sort_by.map(Into::into),
+            order: order_direction.map(Into::into),
+            limit: limit.try_into()?,
+            offset: offset.try_into()?,
+        })?;
+
+        collections
+            .into_iter()
+            .map(TryInto::try_into)
+            .collect::<Result<_, _>>()
+            .map_err(Into::into)
+    }
```

Comment:
This needs to return a new object called CollectionTrend because the UI needs access to all the columns on the table.

CollectionTrend should have a field called collection which is either mcc or non-mcc collection.